data-prepper-logstash-configuration/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'java'
+}
+
+group 'com.amazon'
+version '1.2.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+}
+
+test {
+    useJUnitPlatform()
+}
```

Comment:
You should be able to remove all of this.

Reply:
You can actually remove the `build.gradle` entirely. In a future commit, we can add it back with the new dependencies such as ANTLR.